_i18n/es/resources/user-guides/ledger-wallet-cli.md

```diff
@@ -1,165 +1,165 @@
-2. Open a new terminal
+2. Abre una nueva consola de comandos
 
-3. Now type:
+3. Ahora escribe:
 
-`./monero-wallet-cli --generate-from-device <new-wallet-name> --subaddress-lookahead 3:200`
+`./monero-wallet-cli --generate-from-device <nuevo-nombre-de-monedero> --subaddress-lookahead 3:200`
 
-Note that is simply a placeholder for the actual wallet name. If you, for instance, want to name your wallet `MoneroWallet`, the command would be as follows:
+Ten en cuenta que es simplemente un marcador para el nombre actual del monedero. Si, por ejemplo, deseas llamar a tu monedero `MoneroWallet`, el comando sería como se muestra a continuación:
 
 `./monero-wallet-cli --generate-from-device MoneroWallet --subaddress-lookahead 3:200`
 
-4. The CLI will, after executing aforementioned command, prompt your for a password. Make sure to set a strong password and confirm it thereafter.
+4. La consola te preguntará, después de ejecutar el comando anterior, por una contraseña. Asegúrate de establecer una contraseña fuerte y de confirmarla después de ello.
 
-5. The Ledger will ask whether you want to export the private view key or not. First and foremost, your funds cannot be compromised with merely the private view key. Exporting the private view key enables the client (on the computer - Monero v0.12.2.0) to scan blocks looking for transactions that belong to your wallet / address. If this option is not utilized, the device (Ledger) will scan blocks, which will be significantly slower. There is, however, one caveat. That is, if your system gets compromised, the adversary will potentially be able to compromise your private view key as well, which is detrimental to privacy. This is virtually impossible when the private view key is not exported.
+5. Ledger preguntará si deseas exportar la clave privada o no. Primero y ante todo, tus fondos no pueden ser comprometidos con simplemente tu clave privada de visualización. Exportar la clave privada de visualización permite al cliente (en el ordenador - Monero v0.12.2.0) escanear bloques en busca de transacciones pertenecientes a tu monedero / dirección. Si esta opción no es utilizada, el dispositivo (Ledger) escaneará bloques, lo que será significativamente más lento. Sin embargo, hay una advertencia. Si tu sistema se ve comprometido, el adversario será potencialmente capaz de también comprometer tu clave privada, lo que es perjudicial para tu privacidad. Esto es virtualmente imposible cuando la clave privada no es exportada.
 
-6. You may have to hit confirm twice before it proceeds.
+6. Puedes tener que hacer clic en confirmar dos veces antes de proceder.
 
-7. Your Ledger Monero wallet will now be generated. Note that this may take up to 5-10 minutes. Furthermore, there will be no immediate feedback in the CLI nor on the Ledger.
+7. Tu monedero Ledger Monero será ahora generado. Ten en cuenta que esto puede tomar de 5 a 10 minutos. Además, no habrá realimentación inmediata en la consola de comandos o en la Ledger.
```

Comment:
@BlackLotus64  if you take a look at the website, ["CLI"](https://getmonero.org/es/downloads/) stands for "Command-Line = Linea de Comandos". I suggest to change "consola de comandos" to "línea de comandos" to keep consistency, and when the word "CLI" shows up, either keep "CLI" or "línea de comandos".

Reply:
Does it really say that on the website? CLI is "Command Line *Interface*", not just "Command Line". 
I.e there are two ways to use Monero, on the command line, using the command line interface (CLI), or using the graphical user interface (GUI)

Reply:
Actually I only got the info from the website/download section.

Is better to leave it as CLI and if we want to translate it we should translate it from what it is, "Command Line Interface = Interfaz de Línea de Comandos". Thank you for the clarification @ordtrogen .

Reply:
I'm really not sure about this, at least every person I know just says and knows _consola de comandos_ and I haven't heard or seen someone using _línea de comandos_, they use just the sort version _consola_ for CLI programs, which is also a synonym of _terminal_.

Wikipedia have an article with the name _interfaz de línea de comandos_ and it's just a direct translation of command-line interface, but you know, it's wikipedia, and you cannot just trust everything it says. And for CLI, it may sound OK but not so good because they are an English acronym and it's kinda weird to say _**la CLI**_, I may just use _consola_ for Windows and _terminal_ for Linux and Mac, it sounds more own.

Reply:
I understand your point of view, but CLI is something from the `monero-wallet-cli`. It's hard just to change it because it might sound weird call it "la CLI", "la consola", "la casa", "la televisión", "la computadora", etc.. Or in any case, we should have to use hole name so there won't be any misinterpretations, "Interfaz de línea de comandos(CLI)".
The "CLI"(`monero-wallet-cli`) is the same in Windows, Linux, or Mac. "Terminal" and "cmd" (consola) are something different in the OS, we just can't change the name.

Reply:
If that's the case I agree that it keeps like _interface de línea de comandos_. And yes, it is included in the program but it's not an own name, it just stands for command-line interface, and like I said, you cannot just mix English and Spanish saying _la CLI (la command-line interface)_, it should be translated at least in a way it can be said in the same way, let's suppose that there is an acronym for CLI in Spanish, so it would be _la ILC (la interfaz de línea de comandos)_ or just _la interfaz de línea de comandos_. I'll just use the entire sentence for now.

Reply:
The"files" names can't be changed even if it can be translated. Maybe I didn't add like I did in a comment below, we have to make it clear that it's referring to the CLI.

We can state "la interfaz de linea de comandos **(CLI)**", we have to keep on using "CLI".

Saying "la CLI" is not a grammatical error. If that's the case "la Internet", "los hashes", "la pool", couldn't be used either.